superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FiltersConfigForm/FiltersConfigForm.tsx

```diff
@@ -258,6 +258,19 @@ const StyledAsterisk = styled.span`
   }
 `;
 
+const TimeRangeInfo = styled.div`
```

Comment:
As I assume this can function as a generic filter type info component, this could potentially be renamed to `FilterTypeInfo`

Reply:
👍🏼